File: 0217-contains-duplicate/0217-contains-duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    let counter = 0;
   nums.sort();
    for(let i = 1; i < nums.length; i++)
        {
            if(nums[i] === nums[i -1])
                {
                    return true;
                }
        }
    return false;
}

File: 0217-contains-duplicate/0217-contains-duplicate.cs
public class Solution {
        public bool ContainsDuplicate(int[] nums) {
             var seen = new HashSet<int>(nums.Length); // pre-size to avoid rehashing

 foreach (int num in nums)
 {
     if (seen.Contains(num))
         return true;

     seen.Add(num);
 }

 return false;
        }
}